### DISCUSSION TITLE
Maintainers: what to submit for the assignment

### DISCUSSION THREAD

**Discussion by jtr13** [jtr13/cc22mw#11] [General]
Maintaining the repo counts as your community contribution, but you still have to submit the assignment.

For the CourseWorks submission, list the links of the pull requests that your worked on, such as:

https://github.com/jtr13/cc22mw/pull/1

and complete the motivation / explanation part by describing your experience working on the project, such as what you learned, how the project could be improved, or other relevant observations.

For the GitHub submission, just write "Mananged GitHub repo"

You can take until Dec 1 to complete since you won't be able to work on the pull requests until they are submitted, however it would be helpful to us if you indicated in the assignment comments before the due dates that you are managing the project so it doesn't look to the graders like you didn't complete it.

**Comment by philkim72** [comment]
"For the GitHub submission, just write "Mananged GitHub repo"

Can you clarify where we are supposed to write this?

**Reply by jtr13** [reply]
In the assignment comments for the CC GitHub assignment in CourseWorks. (If not we'll figure it out eventually but it's helpful for us so the graders do not think that you didn't do the assignment.)
